src/components/NotFoundArticle.jsx

```diff
@@ -0,0 +1,13 @@
+import { NcLogo } from "./NcLogo"
+
+function NotFoundArticle() {
```

Comment:
consider making this a reusable component which accepts a prop for what isn't found and then returns it rather than having multiple separate components for essentially the same thing